### ISSUE TITLE
Examples not relevant enough to JS

### ISSUE THREAD

**Issue by mrmanc** [MCR-Digital/apprentice-boot-camp-fundamentals-1#77]
We need to make sure that all the examples shown are relevant to JS. We might choose to do this using modules rather than ES6 classes. There are still examples of Java code in the slides.

> Some of the examples that seemed to be relevant only in Java world, although it was interesting to learn about them anyway

**Comment by helloheatherw** [comment]
I wonder if this might have been related to the parts of the session on inheritance and polymorphism as the example is in Java. Even though the JavaScript versions of these examples won't look hugely different, perhaps it's worth having this on the slides to refer to.

We could also potentially add in a couple of code examples where we discuss OO-CSS and OO in JS.

**Comment by mrmanc** [comment]
Yes, I think having the JS examples added for inheritance and polymorphism will be good.

An example of OO-CSS would certainly bring that concept to life. We should consider whether that slide was valuable though—I’m not sure what I said really added much.
